src/components/Hints.jsx

```diff
@@ -0,0 +1,22 @@
+function Hints() {
+
+    const HINTS = [
+        "Click \"add todo\" to start typing a new task",
+        "Tick the box or click on the task to complete it",
+        "Double-click on the task to edit it"
+    ];
+
+  return (
+    <div className='my-10 flex flex-col items-center'>
+        <div className="text-2xl font-bold text-gray-600 mb-3">Hints</div>
```

Comment:
To stay consistent in the way you're formatting your code, I would suggest sticking with single quotes or double quotes for the className property but not both. It's more readable that way.